sdks/java/io/google-cloud-platform/pom.xml

```diff
@@ -88,11 +88,12 @@
     </plugins>
 
     <extensions>
-      <!-- Use os-maven-plugin to initialize the "os.detected" properties -->
+      <!-- Use os-maven-plugin to initialize the "os.detected" properties for Bigtable.
+           See: https://cloud.google.com/bigtable/docs/using-maven#encryption -->
       <extension>
         <groupId>kr.motd.maven</groupId>
         <artifactId>os-maven-plugin</artifactId>
-        <version>1.4.0.Final</version>
```

Comment:
Because this is an extension and not a plugin, we can't define the version to be used in the `<dependencyManagement>` root pom. Instead, I've added a property definition for the version and reference it here.